runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -520,12 +520,16 @@ private boolean isTarget(NPC npc)
 	private void rebuildTargetNames(Task task)
 	{
 		targetNames.clear();
-		Arrays.stream(task.getTargetNames())
-			.map(String::toLowerCase)
-			.forEach(targetNames::add);
-		targetNames.add(taskName.toLowerCase().replaceAll("s$", ""));
-	}
+		highlightedTargets.clear();
```

Comment:
This line is unnecessary, there's already a `highlightedTargets.clear()` in `rebuildTargetList()` which gets called right after this function